ui/src/app/views/workspace-share/component.spec.ts

```diff
@@ -24,13 +24,19 @@ import {UserRole} from 'generated';
 import {WorkspaceAccessLevel} from 'generated';
 import {WorkspacesService} from 'generated';
 
+interface RoleNamePair {
```

Comment:
nit: calling a 3-property struct a pair seems inaccurate. I would probably call this something like UserRoleEntry or UserRoleRow instead